server/server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const cors = require('cors');
+const dotenv = require('dotenv');
+const PROJECTS = require('./projects');
+
+dotenv.config();
+
+const app = express();
+const PORT = 5000;
+
+app.use(express.json());
+app.use(cors());
+
+const envLogin = process.env.LOGIN;
+const envPassword = process.env.PASSWORD;
+
+app.post('/login', (req, res) => {
+    const { login, password } = req.body;
+
+    if (login === envLogin && password === envPassword) {
+        res.status(200).json({ message: 'Login successful' });
+    } else {
+        res.status(401).json({ message: 'Invalid credentials' });
+    }
+});
+
+app.get('/projects', (req, res) => {
+    const query = req.query.q ? req.query.q.toLowerCase().trim() : '';
```

Comment:
Please avoid creating variables or params as `a, b ,q, a1, ...` and always use meaningful names
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0